src/app/pages/devoluciontributaria/components/show-resumendevolucion/show-resumendevoluciontributaria.component.html

```diff
@@ -0,0 +1,92 @@
+<div class="cotainer">
```

Comment:
Revisar: Quizás ... ?
```suggestion
<div class="container">
```

Reply:
xd